doc/cartridge_kubernetes_guide/index.rst

```diff
@@ -313,52 +313,52 @@ joining a node or creating a replica set. The operator knows how to do
 this better, and if you set the value for its desired system
 configuration, it begins to bring the cluster to the desired state.
 
-The Tarantool Kubernetes operator itself is the implementation of the Kubernetes
+The Tarantool Kubernetes operator itself is an implementation of the Kubernetes
 Operator design pattern. It offers the automation of work with user
 resources using controllers that respond to various events and changes.
 
 The following links can help you understand this pattern:
 
-- `Official description on kubernetes.io <https://kubernetes.io/docs/concepts/extend-kubernetes/operator/>`_
-- `Overview from the creators of the pattern (CoreOS) <https://coreos.com/operators/>`_
+- `Official description on kubernetes.io <https://kubernetes.io/docs/concepts/extend-kubernetes/operator/>`_;
+- `Overview from the creators of the pattern (CoreOS) <https://coreos.com/operators/>`_;
 - `Post on Habr from Lamoda about the development of the operator <https://habr.com/ru/company/lamoda/blog/446648/>`_.
 
 In the meantime, our pod with ``tarantool-operator`` went into a *Running*
 state. The next step is to install the app using the ``tarantool/cartridge``
 helm chart. To do this, prepare a description of the desired system.
 
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-Deploying Tarantool Cartridge application
+Deploying a Tarantool Cartridge application
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 After you have deployed the cluster and installed the operator, you can
 move to the next step--launching the app.
 
 We will deploy the app using the ``tarantool/cartridge`` chart. This is
 a template. Run it with the default settings and get our example application
-that have 3 instances. If you define your own settings, you can deploy any
+that has 3 instances. If you define your own settings, you can deploy any
 application of any topology using the Tarantool Cartridge.
 
 Let's have a look at the settings in the ``values.yaml`` file.
 Comments provide a description of each parameter:
 
-.. code:: yaml
+.. code-block:: yaml
```

Comment:
`code-block` has the "Copy" button, just `code` doesn't.

Reply:
The "copy" button does not appear. It may not be supported with "yaml" format.